docs/antrea-network-policy.md

```diff
@@ -689,6 +690,61 @@ The following kubectl commands can be used to retrieve CG resources:
     kubectl get cg.core.antrea.tanzu.vmware.com
 ```
 
+## Select Namespace by Name
+
+Kubernetes NetworkPolicies and Antrea-native NetworkPolicies allow selecting
+workloads from Namespaces with the use of a label selector (i.e. `namespaceSelector`).
+However, it is often desired that the Namespaces be selected directly by its `name`
```

Comment:
it is often desirable to be able to select Namespaces directly using their `name` as opposed...

Reply:
done